src/Token.php

```diff
@@ -81,39 +72,22 @@ public function test($type, $values = null)
         );
     }
 
-    /**
-     * @return int
-     */
-    public function getLine()
+    public function getLine(): int
     {
         return $this->lineno;
     }
 
-    /**
-     * @return int
-     */
-    public function getType()
+    public function getType(): int
     {
         return $this->type;
     }
 
-    /**
-     * @return string
```

Comment:
the return type needs to be documented

Reply:
I've removed it as this is mixed.

Reply:
is it actually mixed, or `scalar|null` which is more precise ?

And then, documenting it is still better, as it tells people what this indeed is (plus, we need to update the phpdoc in older branch if it is wrong when saying `string`)